src/pages/index.md

```diff
@@ -2,6 +2,32 @@
 template: home-page
 path: /
 hero:
-  title: Test title
-  body: Test body
+  title: Threshold powers user sovereignty on the public blockchain.
+  body: Access cryptographic tools that ensure full control over your digital assets.
+  ctaButtons:
+    - label: Start Staking
+      url: https://dashboard.threshold.network
+    - label: Read the Blog
+      url: https://blog.threshold.network/
+summary:
+  title: Why Threshold Network?
+  body: Threshold Network is the first ever on-chain merge between two existing
+    networks. <a href="https://blog.threshold.network/" target="_blank"
```

Comment:
Is there a way to use our enums for external links with the CMS? Would be nice to be able to reference a variable instead of having random strings in case things change. I also like having the enums because it's obvious which links we are still missing from product.

Reply:
I don't think so but need to dig into it. 

> in case things change.

I think that's why we have cms- so any non-technical person can update the invalid link via user-friendly CMS UI instead of making changes in source code + git magic.